### PR TITLE
(feat) connect market stats to real data

### PR DESCRIPTION
**Pull request by pepopowitz**
This PR connects the market stats module to real data. It currently shows only the stats for the first medium, but in the future we'll be able to pick which medium we want to see.

My Progress: 
- [x] MarketStats extracted to its own component & wrapped with a FragmentContainer
- [x] Relay hooked up to provide data
- [x] MarketStats emitting real data just to prove connectivity. 
- [x] See what it looks like when I waterfall/nest QueryRenderers, so that the priceInsights query isn't fetched until the artist returns with internalID. It looks fine because the nested QueryRenderer is on a different tab when it renders!
- [x] Fill in all the correct data.
- [x] Refactor the `formatText` util to use the new `formatLargeNumber` util introduced in this PR

For followup PRs: 
- [ ] Add the ability to select a medium from a `<Picker>`. 
- [ ] Add a loading skeleton
- [ ] Tests

The type of this PR is: **Enhancement**

This PR partially addresses [CX-738]

### Description

Connects real data to the market stats module. 

How it looks:

simulator_screenshot_491F4899-4F33-4E70-82D6-B057EC5CB760

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [ ] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-738]: https://artsyproduct.atlassian.net/browse/CX-738

### CODE REVIEW

File: src/lib/Components/Artist/ArtistInsights/MarketStats.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        </Text>
+        <TouchableOpacity hitSlop={{ top: 20, right: 20, bottom: 20, left: 20 }}>
+          <Image source={require("@images/info.png")} />
+        </TouchableOpacity>
+      </Flex>
+      <Text variant="small" color="black60">
+        Based on the last 12 months of auction data
+      </Text>
+      {priceInsights.edges?.map((e) => {
+        return (
```
Comment (pepopowitz) [inline src/lib/Components/Artist/ArtistInsights/MarketStats.tsx:26]: This is just to prove connectivity.

**Comment by pepopowitz** [comment]
I've got this PR rebased on master, and I intend to follow up with the last remaining tasks in future PRs. 

I've got a broken test for the artist page - the error is pretty cryptic, but I suspect it is due to the fact that I've added a nested QueryRenderer: 

```
FAIL  src/lib/Scenes/Artist/__tests__/Artist-tests.tsx
  ● availableTabs › returns two tabs if artist has metadata, works, and shows when AROptionsNewInsightsPage is true

    Failed: "TypeError: Cannot read property '_responseForQueryIDWithVariables' of undefined"

```

I'll take a look at this later today or Monday, but if anyone from @artsy/collector-experience has thoughts about that error, I'm all ears.

**Comment by brainbicycle** [comment]
Hmmm the `yarn bundle-for-native-ci` command seems to not like the underscores in the numbers in `formatLargeNumber` wonder if this is typescript version delta? removing the underscores seems to fix it

File: src/lib/utils/formatLargeNumber.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export function formatLargeNumber(number: number, decimalPlaces: number = 0) {
+  if (number < 1_000) {
```
Comment (brainbicycle) [inline src/lib/utils/formatLargeNumber.ts:2]: the `yarn bundle-for-native-ci` command on circle seems to not like the underscores in the numbers in this function wonder if this is typescript version delta? removing the underscores seems to fix it

File: src/lib/utils/__tests__/formatLargeNumberOfItems-tests.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        `"850 works for sale"`
+      )
+    })
+    it("formats more than a thousand less than a million", () => {
+      expect(formatLargeNumberOfItems(151000, "work for sale", "works for sale")).toMatchInlineSnapshot(
+        `"151.0k works for sale"`
+      )
+    })
+    it("formats more than a million", () => {
+      expect(formatLargeNumberOfItems(2200000, "work for sale", "works for sale")).toMatchInlineSnapshot(
```
Comment (brainbicycle) [inline src/lib/utils/__tests__/formatLargeNumberOfItems-tests.ts:34]: Nice tests!

File: src/lib/utils/formatLargeNumber.ts (hunk 2 above)
Comment (pepopowitz) [reply src/lib/utils/formatLargeNumber.ts:2]: ah, cool, I was surprised I didn't get a linting error anyway. I'll remove them!

**Review by brainbicycle — APPROVED** [review]
Looks great 📈 !

File: src/lib/utils/formatLargeNumber.ts (hunk 2 above)
Comment (pepopowitz) [reply src/lib/utils/formatLargeNumber.ts]: though this code really shows the value of `_` separators in large numbers.

**Pull request merged**